capstone/src/main/java/com/google/univiz/UniversityData.java

```diff
@@ -0,0 +1,67 @@
+package com.google.univiz;
+
+import com.google.auto.value.AutoValue;
+
+@AutoValue
+public abstract class UniversityData {
+
+  public static UniversityData create(
+      String name,
+      String city,
+      float latitude,
+      float longitude,
+      int flagMainCampus,
+      int urbanizationDegree,
+      float admissionRate,
+      float avgSAT,
+      int numOfUndergrads,
+      int avgCost,
+      int studentsInDebt,
+      int carnegieSizeDegree,
```

Comment:
carnegieSizeDegree doesn't mean much to the causal reader. also would benefit from javadoc.